### ISSUE TITLE
List.insert makes the inserted object into an array in IE 11

### ISSUE THREAD

**Issue by Methuselah96** [immutable-js-oss/immutable-js#135]
*From @bowen0 on Tue, 21 May 2019 10:56:08 GMT*

### What happened

List.insert makes the inserted object into an array in IE 11 

IE version : 11.648.17134.0

I use IE 11 to open https://immutable-js.github.io/immutable-js/docs/#/List to test in the console.

### How to reproduce
1. Defined an object list
2. Insert an object at the beginning of the list 
3. The first object becomes an array

e.g.
```
var a= Immutable.fromJS({ a: 0 })
var b = Immutable.fromJS({ b: 0 })
var c = Immutable.fromJS({ c: 0 });
var list = Immutable.fromJS([b, c])

var newlist = list.insert(0, a); 
// expect [Object, Object, Object]
// actual:  [Array, Object, Object]
```
pic 
![image](https://user-images.githubusercontent.com/22339946/58090578-14a66380-7bfa-11e9-8499-7afef780c64b.png)

*Copied from original issue: https://github.com/immutable-js/immutable-js/issues/1711*

**Comment by Methuselah96** [comment]
Likely fixed by https://github.com/immutable-js-oss/immutable-js/pull/194, but should be tested.

**Comment by jdeniau** [comment]
Fixed in master by #194 

On immutable website (4.0.0-rc.12)
![image](https://user-images.githubusercontent.com/1398469/100517278-ce847580-3189-11eb-9fe5-8fc92cf8d46e.png)

With the build on master
![image](https://user-images.githubusercontent.com/1398469/100517285-d93f0a80-3189-11eb-8e4d-efa80e49e967.png)

**Comment by Methuselah96** [comment]
Thanks!
